Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -23,18 +23,24 @@
     sys.exit(1)
 
 ami_client = None
-PHYS_VALUES = [{"ami": "genFiltEff", "es": "gen_filt_eff"},
-               {"ami": "crossSection", "es": "cross_section"},
-               {"ami": "crossSectionRef", "es": "cross_section_ref"},
-               {"ami": "kFactor", "es": "k_factor"},
-               {"ami": "processGroup", "es": "process_group"},
-               {"ami": "mePDF", "es": "me_pdf"},
+
+# Field names in terms of AMI and ES schemes.
+PHYS_VALUES = [{'ami': 'genFiltEff', 'es': 'gen_filt_eff'},
+               {'ami': 'crossSection', 'es': 'cross_section'},
+               {'ami': 'crossSectionRef', 'es': 'cross_section_ref'},
+               {'ami': 'kFactor', 'es': 'k_factor'},
+               {'ami': 'processGroup', 'es': 'process_group'},
+               {'ami': 'mePDF', 'es': 'me_pdf'},
                ]
 FILTER = ['AOD', 'EVNT', 'HITS']
 
 
 def main(argv):
-    """ Main program body. """
+    """ Main program body.
+
+    :param argv: arguments
```

Comment:
Not very explicative description.. mb "command line arguments"? I mean, it's pretty obvious that function arguments are arguments, so this line looks like "this line must be here due to some stupid requirements, so it's here -- but no one said it must make any sense: as I said, the requirements are stupid."

Reply:
Done.